src/docs/product/metrics/metrics-explorer-page.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Metrics Explorer Page"
+sidebar_order: 12
+description: "Learn about how you can visualise and explore your metrics in the Sentry UI."
```

Comment:
```suggestion
description: "Learn how to visualize and explore your data metrics that are important to you, including latency, processing time, event size, user signups, and conversion rates."
```

Reply:
Those are just examples of custom metrics one might create, so I wouldn't include them in the high 

```suggestion
description: "Learn how to visualize and explore your application's performance metrics together with custom metrics you send to Sentry, by using the Metrics page."
```